tests/run-make/static-pie/rmake.rs

```diff
@@ -0,0 +1,48 @@
+// How to manually run this
+// $ ./x.py test --target x86_64-unknown-linux-[musl,gnu] tests/run-make/static-pie
+
+//@ only-x86_64
+//@ only-linux
+//@ ignore-32bit
+
+use std::process::Command;
+
+use run_make_support::llvm_readobj;
+use run_make_support::rustc;
+use run_make_support::{cmd, env_var};
+
+fn ok_compiler_version(compiler: &str) -> bool {
+    let check_file = format!("check_{compiler}_version.sh");
+
+    Command::new(check_file).status().is_ok_and(|status| status.success())
```

Comment:
Suggestion: I think you can use `cmd(check_file).run()` (from the support lib)

Reply:
`cmd(check_file).run()` panics if the command fails. In this case, the command failing means that the compiler was not found and it should be skipped.

Reply:
Wait, `cmd(check_file).run().status().success()` might work. I will port the shell scripts though, so this code won't be used anymore.

Reply:
Ah yes you're correct, it would need to be something like `cmd(check_file).run_unchecked().status().success()` to preserve the original logic.